README.md

```diff
@@ -1 +1,9 @@
-# hw01
\ No newline at end of file
+# HW 01
```

Comment:
Standard R Markdown practice is to have a return after each markdown element (headers, code chunks, plots etc.). That'll help space things out and will help you avoid any rendering issues later on.